README

```diff
@@ -132,15 +132,10 @@ CD-ROM: My CD-ROM doesn't work.
     in Linux:
       - mount d /media/cdrom -t cdrom
```

Comment:
The distinction now is that `/media/cdrom` needs to be a filesystem as opposed to block device (and `/media/cdrom` is a bit ambiguous now because some distro's those are aliases to the devices like `/dev/sr0`).

Suggest something like: mount d /path/to/cdrom -t cdrom

That way, the user can mount their CD-ROM host-side and DOSBox can still access it "directly" (albeit without CD-DA support).

Reply:
Many distros standardized on using `/run/media/<user>/<drive_label>` now, but I agree - I'll replace it with `/path/to/cdrom`.